test/views/WhatYouOweViewSpec.scala

```diff
@@ -89,7 +92,8 @@ class WhatYouOweViewSpec extends TestSupport with FeatureSwitching with Implicit
               dunningLock: Boolean = false,
               whatYouOweCreditAmountEnabled: Boolean = false,
               migrationYear: Int = LocalDate.now().getYear - 1,
-              codingOutEnabled: Boolean = true
+              codingOutEnabled: Boolean = true,
+              MFADebitsEnabled: Boolean = true
```

Comment:
this isnt really doing anything since we never set it to false

Reply:
Removed in latest commit